utils/print_qor.py

```diff
@@ -78,23 +77,32 @@ def main():
     )
     w = csv.DictWriter(sys.stdout, fields)
     w.writeheader()
+    rows = list()
     for root, dirs, files in os.walk(args.build_dir):
         if 'pack.log' in files:
             d = {}
 
-            d['path'] = root
-            d['pack time (sec)'] = scan_runtime(os.path.join(root, 'pack.log'))
+            path = root.split('/')
```

Comment:
Factor this into a function like `last_n_dirs(root, 2)`, etc.

Reply:
Done